src/components/FormDropdown.tsx

```diff
@@ -5,7 +5,7 @@ import { DropdownOption } from '../types/ui'
 import Dropdown from './Dropdown'
 
 type Props = {
-  options: DropdownOption[]
+  options: () => DropdownOption[]
```

Comment:
Should this use `Accessor`?

```suggestion
  options: Accessor<DropdownOption[]>
```